### PR TITLE
Update Style of the Search Box on the Landing Page

### PR DESCRIPTION
**Pull request by Catherinesjkim**
Locations:
- SearchForm.js

Details:
1. Updated the search/input box to make it look more presentable for the stakeholder meeting tonight.

### CODE REVIEW

**Review by JeffreyAsuncion — DISMISSED** [review]
Great work. Keep it up.

**Review by kimly143 — DISMISSED** [review]
Thanks for fixing the styles

**Review by JeffreyAsuncion — APPROVED** [review]
Great work. Keep it up.

**Review by vetorobert3 — DISMISSED** [review]
Catherine, I see the fix for that annoying limit warning. Great job! also noticed the webpack dependency and various changes to styling.

**Review by vetorobert3 — COMMENTED** [review]
I noticed the failed test. Says not to worry about the babel dependency, because it will update itself. I'll take it's word for it.

**Comment by Catherinesjkim** [comment]
> I noticed the failed test. Says not to worry about the babel dependency, because it will update itself. I'll take it's word for it.

Thanks! I think you need to approve it for me to merge it...

**Review by JeffreyAsuncion — DISMISSED** [review]
Awesome. Final stretch.

**Comment by tinomen** [comment]
This PR is humungous. Did anyone actually review the changes?

File: amplify.yml [hunk 1]
```diff
@@ -1,19 +0,0 @@
-version: 1
```
Comment (tinomen) [inline amplify.yml]: Why on earth is this file being removed?

**Review by tinomen — CHANGES_REQUESTED** [review]
You have a problem with the package.json and package-lock.json files. I suggest grabbing the files from the main branch and re-install your new modules

File: amplify.yml (hunk 1 above)
Comment (tinomen) [reply amplify.yml]: This file needs to be restored so it doesn't break the aws amplify deploy

Comment (Catherinesjkim) [reply amplify.yml]: - Restored amplify.yml file so it doesn't break the aws amplify deployment
- Grabbed the package.json from the main branch and re-installed the new modules.

**Review by JeffreyAsuncion — DISMISSED** [review]
Keep up the Hard work.

File: src/components/common/SearchForm.js [hunk 2]
```diff
@@ -14,10 +14,7 @@ const ColStyle = {
 };
 
 const SearchStyle = {
-  width: '80vw',
-  minWidth: '24rem',
-  maxWidth: '80rem',
-  padding: '1rem',
+  width: '600p',
 };
```
Comment (kimly143) [inline src/components/common/SearchForm.js]: 600p ? 
p is not a unit.

**Review by kimly143 — COMMENTED** [review]
I think you make a typo on style.

**Review by JeffreyAsuncion — DISMISSED** [review]
Way to go. Keep up the Hard work.

**Review by kimly143 — DISMISSED** [review]

File: src/components/common/SearchForm.js (hunk 2 above)
Comment (Catherinesjkim) [reply src/components/common/SearchForm.js]: Just updated the typo.

**Review by kimly143 — DISMISSED** [review]

**Review by tinomen — APPROVED** [review]
It's tough when adding new npm modules and the package-lock file has a crazy amount of changes. You can give a warning on the PR description when that happens.

**Review by kimly143 — APPROVED** [review]
Style changed and updated package json

**Pull request merged**